requirements/docs.txt

```diff
@@ -0,0 +1,78 @@
+#
```

Comment:
The file above is manually created, but this one is created by running `hdev requirements`

Reply:
It would be great if the instructions at the top of this file could be updated to reflect the current process.

Reply:
We actually can't do that unfortunately without creating problems with Dependabot. We both use `pip-compile` to create this file I think. So if we change the header Dependabot will change it back. This means a lot of PRs would have noise adding and removing a custom header.